lib/services/base.js

```diff
@@ -1,13 +1,18 @@
 const Stripe = require('stripe');
 
 module.exports = class BaseService {
-  constructor (options = {}) {
-    if (!options.secretKey) {
+  constructor (secretKey, config = {}) {
+    // If first parameter is an object, we assume the old behavior of passing in secretKey
+    if(typeof secretKey === 'object') {
+      secretKey = secretKey.secretKey;
+      this.paginate = secretKey.paginate;
```

Comment:
the order of these should be switched, overwriting secretKey should be last.

I'd recommend setting config.pagination here instead